templates/admin/dashboard.tmpl

```diff
@@ -76,7 +76,7 @@
 			{{ctx.Locale.Tr "admin.dashboard.system_status"}}
 		</h4>
 		{{/* TODO: make these stats work in multi-server deployments, likely needs per-server stats in DB */}}
-		<div hx-get="{{$.Link}}/system_status" hx-swap="morph:innerHTML" hx-trigger="every 5s" hx-indicator=".divider" class="ui attached table segment">
+		<div hx-get="{{$.Link}}/system_status" hx-swap="morph:innerHTML" hx-trigger="every 5s" hx-indicator=".tw-hidden" class="ui attached table segment">
```

Comment:
`hx-indicator` means add the loading indicator to the selector.

So, just use a non-existing selector for it. eg:


```suggestion
		<div hx-get="{{$.Link}}/system_status" hx-swap="morph:innerHTML" hx-trigger="every 5s" hx-indicator=".no-such-indicator" class="ui attached table segment">
```

Use any explicit name to tell the readers that "we do not need to match any element to add indicator"

It is more intuitive when reading.

Reply:
Actually it could also use something like `hx-indicator="no-indicator"` ,  because `no-indicator` also matches nothing.

Reply:
But that prints an error in the console on every request

Reply:
I see, then `tw-hidden` also doesn't seem good enough because `tw-hidden` might also not exist.


Ideally it could be like this:
```suggestion
		<div hx-get="{{$.Link}}/system_status" hx-swap="morph:innerHTML" hx-trigger="every 5s" hx-indicator=".hidden-loading-indicator" class="ui attached table segment">
			<div class="hidden-loading-indicator tw-hidden"></div>
```

Then everything is clear and I think it reads better.

Reply:
And more details, `.tw-hidden` can't be right. It just adds the indicator to all the `tw-hidden` elements on this page, which causes various side effects

Think about a case: 

```
<div class="panel tw-hidden"></div>
<div hx-get hx-indicator=".tw-hidden"></div>
```

The "panel" is not related to the hx-get.

When the loading-indicator is added to the "panel", and the request is still running, then the user shows the panel: surprise: they sees a "loading panel".